build_wasm.sh

```diff
@@ -1,6 +1,5 @@
 PROJECT_NAME=weblibdedx
-PROPER_PATH="\`\${process.env.PUBLIC_URL}\/$PROJECT_NAME.wasm\`"
-BROKEN_PATH="wasmBinaryFile = \'weblibdedx.wasm\'"
+FIXED_PATH="\${process.env.PUBLIC_URL}\/$PROJECT_NAME.wasm"
```

Comment:
What is ${process.env.PUBLIC_URL} ? I doubt that `build_wasm` would see the github action specific env variables.

I'd extract manipulation of this type completely from `build_wasm.sh` and move it to deploy stage (to github actions).

Imagine somebody wants to use this script in a cloned repository.

Reply:
Mind the "\" before the $. What it does is it changes the code to have that line in it. The value is read later when the code is run through react itself

Reply:
In other words the $ sign in this case is ment to be used as JavaScript string interpolation.

Reply:
Also take into account that the choice of the env variable isn't random - it's the same that's later responsible for loading the manifest.json and all assets in index.html (in the deployed builds)

Reply:
OK, its clear now